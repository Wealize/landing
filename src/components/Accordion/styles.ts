import styled from 'styled-components'

import { sixthPx, halfPx, twoThirdPx, singlePx, singleAndHalfPx, doublePx, doubleAndHalfPx, triplePx, quadruplePx, quadruple } from '../../theme/space'
import { media } from '../../theme/media'

export const Container = styled.div<{ backgroundColor: string, textColor: string}>`
  width: 100%;
  min-height: 20vh;
  padding: ${doubleAndHalfPx()} ${singlePx()};

  ${media.greaterThan('md')`
    padding: ${triplePx()} ${quadruplePx()};
  `};

  ${media.greaterThan('lg')`
    padding: ${quadruplePx()} ${quadruple() * 1.5}px;
  `};

  ${media.greaterThan('xl')`
    padding: ${quadruplePx()} ${quadruple() * 2.5}px;
  `};

  ${media.greaterThan('xxl')`
    padding: ${quadruplePx()} ${quadruple() * 3.5}px;
  `};

  color: ${({ textColor }) => textColor};
  background-color: ${({ backgroundColor }) => backgroundColor};
`

export const TitleContainer = styled.div`
  padding-bottom: 0;

  > .accordion-separator{
    margin-top: ${halfPx()};
  }
`

export const Title = styled.h5<{ textColor: string}>`
  font-size: 1rem;
  text-transform: uppercase;
  font-weight: bold;
  color: ${({ textColor }) => textColor};

  ${media.greaterThan('xl')`
    font-size: 1.1rem;
  `};

  ${media.greaterThan('lg')`
    font-size: 1.2rem;
  `};

  ${media.greaterThan('xxl')`
    font-size: 1.2rem;
  `};
`

export const Separator = styled.hr<{ textColor: string}>`
  width: 100%;
  background-color: ${({ textColor }) => textColor};
  opacity: .6;
  border: none;
  height: 2px;
  margin: 0;
`

export const SectionContainer = styled.div`
  padding-top: ${singleAndHalfPx()};

  > .accordion-separator {
    margin-top: ${singleAndHalfPx()};
  }

  ${media.greaterThan('md')`
    padding-top: ${doublePx()};

    > .accordion-separator {
      margin-top: ${doublePx()};
    }
  `};
`

export const SectionHeader = styled.div<{isExpanded: boolean}>`
  display: flex;
  width: 100%;
  flex-direction: row;
  justify-content: space-between;
  align-items: center;
  padding-bottom: 0;

  > .button-rounded {
    display: none;

    & span {
      font-weight: lighter;
    }
  }


  ${media.greaterThan('lg')`
    > .accordion-section-toggle-button {
      display: none;
    }

    > .button-rounded {
      display: block;
      min-height: 75px;
    }
  `};

`

export const SectionTitle = styled.h6`
  font-size: 1.7rem;
  width: 75%;
  line-height: 1.52;

  ${media.greaterThan('md')`
    font-size: 1.8rem;
  `};

  ${media.greaterThan('lg')`
    font-size: 2.2em;
  `};

  ${media.greaterThan('xl')`
    font-size: 2.4rem;
  `};

  ${media.greaterThan('xxl')`
    font-size: 2.7rem;
  `};
`

export const SectionToggle = styled.button<{
  textColor: string
  backgroundColor: string
  isExpanded: boolean
}>`
  width: 40px;
  height: 40px;
  background-color: transparent;
  border: none;
  padding: 0;
  transition: all 0.5s ease-out;
  transform: ${({ isExpanded }) =>
    isExpanded ? 'rotate(0deg)' : 'rotate(180deg)'};

  & svg > g:first-child {
    fill: ${({ textColor, isExpanded }) => isExpanded ? textColor : 'none'};
  }

  & svg > g > path:last-child {
    fill: ${({ textColor }) => textColor};
  }

  & svg > path {
    fill: ${({ textColor, backgroundColor, isExpanded }) => isExpanded ? backgroundColor : textColor};
  }

  ${media.greaterThan('md')`
      width: 50px;
      height: 50px;
  `};

  ${media.greaterThan('lg')`
    display: none;
  `};
`

export const SectionContentContainer = styled.div<{isExpanded: boolean}>`
  width: 100%;
  display: grid;
  grid-template-columns: repeat(2, 1fr);
  grid-column-gap: ${singlePx()};
  transition: all 0.5s ease-out;
  overflow: hidden;
  height: auto;
  max-height: ${({ isExpanded }) => (isExpanded ? 1000 : 0)}px;
  padding-top: ${({ isExpanded }) => (isExpanded ? singleAndHalfPx() : 0)};

  ${media.greaterThan('lg')`
    grid-template-columns: repeat(3, 1fr);
  `};

  ${media.greaterThan('xl')`
    grid-template-columns: repeat(3, 1fr);
  `};

  ${media.greaterThan('xxl')`
    grid-template-columns: repeat(4, 1fr);
  `};

`

export const SectionElement = styled.span`
  padding: ${twoThirdPx()} 0 ${sixthPx()};
  line-height: 1.4;

  &:before {
    content: '- '
  }

  ${media.greaterThan('md')`
    font-size: 1.2rem;
    padding: ${twoThirdPx()};
  `};

  ${media.greaterThan('lg')`
    font-size: 1.3rem;
  `};

  ${media.greaterThan('xl')`
    font-size: 1.4rem;
  `};

  ${media.greaterThan('xxl')`
    font-size: 1.5rem;
  `};
`
