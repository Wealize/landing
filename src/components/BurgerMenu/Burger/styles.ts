import styled from 'styled-components'

import { ACCENT_COLOR } from '../../../theme/color'
import { media } from '../../../theme/media'
import { singlePx } from '../../../theme/space'

export const StyledBurger = styled.button<{ open: boolean }>`
  top: ${singlePx()};
  margin-top: 2.5%;
  left: ${({ open }) => (open ? '85%' : '85%')};
  width: 2rem;
  height: 1.5rem;
  position: fixed;
  background: transparent;
  z-index: 10;
  display: flex;
  flex-direction: column;
  justify-content: space-around;
  border: none;
  cursor: pointer;
  padding: 0;
  &:focus {
    outline: none;
  }

  ${media.greaterThan('md')`
    margin-top: 0.4%;
  `}

  ${media.greaterThan('lg')`
    display: none;
  `}

  div {
    width: 1.5rem;
    height: 0.18rem;
    border-radius: 10px;
    transition: all 0.3s linear;
    position: relative;
    transform-origin: 1px;
    background-color: ${ACCENT_COLOR};

    :first-child {
      transform: ${({ open }) => (open ? 'rotate(45deg)' : 'rotate(0)')};
    }
    :nth-child(2) {
      opacity: ${({ open }) => (open ? '0' : '1')};
      transform: ${({ open }) => (open ? 'translateX(20px)' : 'translateX(0)')};
    }
    :nth-child(3) {
      transform: ${({ open }) => (open ? 'rotate(-45deg)' : 'rotate(0)')};
    }
  }
`
