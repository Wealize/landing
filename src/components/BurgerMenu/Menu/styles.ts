import styled from 'styled-components'

import {
  ACCENT_COLOR,
  DARK_GRAY_COLOR,
  WHITE_COLOR
} from '../../../theme/color'
import { media } from '../../../theme/media'

export const StyledMenu = styled.nav<{ open: boolean }>`
  position: fixed;
  top: 0;
  left: 0;
  height: 100%;
  width: 100vw;
  padding-left: 2em;
  padding-top: 20vh;
  background-color: ${WHITE_COLOR};
  display: flex;
  flex-direction: column;
  z-index: 5;
  transition: transform 0.3s ease-in-out;
  transform: ${({ open }) => (open ? 'translateY(0%)' : 'translateY(-100%)')};

  ${({ open }) => media.greaterThan('md')`
    transform: ${open ? 'translateX(65%)' : 'translateX(100%)'};
    padding-top: 14vh;
  `}

  ${media.greaterThan('lg')`
    display: none;
  `}
`

export const StyledLink = styled.a<{ isCurrentPage: boolean }>`
  font-weight: bold;
  text-decoration: none;
  color: ${DARK_GRAY_COLOR};
  font-size: 1.125rem;
  cursor: pointer;
  margin-bottom: 32px;
  max-width: fit-content;
  border-bottom: 3px solid
    ${({ isCurrentPage }) => (!isCurrentPage ? 'transparent' : ACCENT_COLOR)};

  transition: border-bottom 0.6s;

  & :hover {
    border-bottom: 3px solid ${ACCENT_COLOR};
  }
`
