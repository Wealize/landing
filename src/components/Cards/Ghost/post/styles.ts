import styled from 'styled-components'

import { WHITE_COLOR } from '../../../../theme/color'
import { singlePx } from '../../../../theme/space'

export const Container = styled.a`
  width: 100%;
  background: ${WHITE_COLOR};
  z-index: 1;
`

export const CardHeadline = styled.h4`
  font-size: 1rem;
  font-weight: bold;
  padding-bottom: ${singlePx()};
`

export const CardPublishedAt = styled.span`
  font-size: 1rem;
  padding: ${singlePx()} 0;
`
