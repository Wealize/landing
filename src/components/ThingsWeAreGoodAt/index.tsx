import React from 'react'
import { useInView } from 'react-intersection-observer'
import useTranslation from 'next-translate/useTranslation'

import CardExpandable from '../Cards/Expandable'
import BlockchainIcon from '../icons/thingsWeAreGoodAt/BlockchainIcon'
import CognitiveIcon from '../icons/thingsWeAreGoodAt/CognitiveIcon'
import ConversationalIcon from '../icons/thingsWeAreGoodAt/ConversationalIcon'
import {
  THINGS_WE_ARE_GOOD_AT_FIRST_CARD_HEADLINE,
  THINGS_WE_ARE_GOOD_AT_FIRST_CARD_SUBHEADLINE,
  THINGS_WE_ARE_GOOD_AT_FIRST_CARD_DESCRIPTION,
  THINGS_WE_ARE_GOOD_AT_SECOND_CARD_HEADLINE,
  THINGS_WE_ARE_GOOD_AT_SECOND_CARD_SUBHEADLINE,
  THINGS_WE_ARE_GOOD_AT_SECOND_CARD_DESCRIPTION,
  THINGS_WE_ARE_GOOD_AT_THIRD_CARD_HEADLINE,
  THINGS_WE_ARE_GOOD_AT_THIRD_CARD_SUBHEADLINE,
  THINGS_WE_ARE_GOOD_AT_THIRD_CARD_DESCRIPTION
} from '../../data/thingsWeAreGoodData'
import {
  PRIMARY_COLOR,
  SECONDARY_COLOR,
  WHITE_COLOR,
  PRIMARY_LIGHT_COLOR,
  ACCENT_COLOR
} from '../../theme/color'

import {
  Container,
  Title,
  CardsContainer,
  FirstCard,
  SecondCard,
  ThirdCard
} from './styles'

const ThingsWeAreGoodAt = () => {
  const { ref, entry } = useInView({
    threshold: 0,
    triggerOnce: true
  })

  const { t } = useTranslation('profile')

  return (
    <Container ref={ref}>
      <Title>{t('things-we-are-good-at.title')}</Title>
      <CardsContainer>
        <FirstCard isVisible={entry?.isIntersecting}>
          <CardExpandable
            headline={t('things-we-are-good-at.first-card.headline', { text: THINGS_WE_ARE_GOOD_AT_FIRST_CARD_HEADLINE })}
            subheadline={t('things-we-are-good-at.first-card.subheadline', { text: THINGS_WE_ARE_GOOD_AT_FIRST_CARD_SUBHEADLINE })}
            description={t('things-we-are-good-at.first-card.description', { text: THINGS_WE_ARE_GOOD_AT_FIRST_CARD_DESCRIPTION })}
            image={<BlockchainIcon />}
            backgroundColor={PRIMARY_LIGHT_COLOR}
            textColor={PRIMARY_COLOR}
            isExpanded={true}
          />
        </FirstCard>

        <SecondCard isVisible={entry?.isIntersecting}>
          <CardExpandable
            headline={t('things-we-are-good-at.second-card.headline', { text: THINGS_WE_ARE_GOOD_AT_SECOND_CARD_HEADLINE })}
            subheadline={t('things-we-are-good-at.second-card.subheadline', { text: THINGS_WE_ARE_GOOD_AT_SECOND_CARD_SUBHEADLINE })}
            description={t('things-we-are-good-at.second-card.description', { text: THINGS_WE_ARE_GOOD_AT_SECOND_CARD_DESCRIPTION })}
            image={<CognitiveIcon />}
            backgroundColor={PRIMARY_COLOR}
            textColor={WHITE_COLOR}
            isExpanded={false}
          />
        </SecondCard>

        <ThirdCard isVisible={entry?.isIntersecting}>
          <CardExpandable
            headline={t('things-we-are-good-at.third-card.headline', { text: THINGS_WE_ARE_GOOD_AT_THIRD_CARD_HEADLINE })}
            subheadline={t('things-we-are-good-at.third-card.subheadline', { text: THINGS_WE_ARE_GOOD_AT_THIRD_CARD_SUBHEADLINE })}
            description={t('things-we-are-good-at.third-card.description', { text: THINGS_WE_ARE_GOOD_AT_THIRD_CARD_DESCRIPTION })}
            image={<ConversationalIcon />}
            backgroundColor={SECONDARY_COLOR}
            textColor={ACCENT_COLOR}
            isExpanded={false}
          />
        </ThirdCard>
      </CardsContainer>
    </Container>
  )
}

export default React.memo(ThingsWeAreGoodAt)
