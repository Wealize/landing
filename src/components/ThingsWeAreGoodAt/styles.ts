import styled from 'styled-components'

import { ACCENT_COLOR, WHITE_COLOR } from '../../theme/color'
import {
  singlePx,
  doublePx,
  triplePx,
  quadruplePx,
  triple,
  quadruple
} from '../../theme/space'
import { media } from '../../theme/media'
import { DEFAULT_RADIUS } from '../../theme/border'

export const Container = styled.div`
  width: 100%;
  background-color: ${ACCENT_COLOR};
  padding-top: ${singlePx()};
  border-top-right-radius: ${DEFAULT_RADIUS}px;

  ${media.greaterThan('md')`
    padding-top: ${doublePx()};
  `};

  ${media.greaterThan('lg')`
    position: relative;
    width: 90%;
    border-top-right-radius: initial;
    border-top-left-radius: ${DEFAULT_RADIUS}px;
    margin: ${triplePx()} 0 ${triple() * 7}px 10%;
  `};

  ${media.greaterThan('xl')`
    margin: ${quadruplePx()} 0 ${quadruple() * 6}px 10%;
  `};

  ${media.greaterThan('xxl')`
    margin: ${quadruplePx()} 0 ${quadruple() * 8}px 10%;
  `};
`

export const Title = styled.h3`
  width: 100%;
  background-color: ${ACCENT_COLOR};
  padding: 0 ${singlePx()} ${quadruplePx()};
  color: ${WHITE_COLOR};
  line-height: 1.33;
  font-weight: bold;
  text-transform: uppercase;

  ${media.greaterThan('md')`
    font-size: 1.3rem;
    padding-left: ${doublePx()};
    padding-bottom: ${triplePx()};
  `};

  ${media.greaterThan('lg')`
    padding: ${doublePx()} ${doublePx()} ${quadruplePx()};
    font-size: 1.5rem;
  `};

  ${media.greaterThan('xl')`
    font-size: 2rem;
  `};
`

export const CardsContainer = styled.div`
  width: 100%;
  background-color: ${ACCENT_COLOR};
  transition: all 0.3s;

  ${media.greaterThan('lg')`
    display: flex;
    flex-direction: row;
    min-height: 500px;
  `};

  ${media.greaterThan('xl')`
    display: flex;
    flex-direction: row;
  `};
`

export const FirstCard = styled.article<{
  isVisible: boolean
}>`
  ${({ isVisible }) => media.greaterThan('lg')`
    transition: all 1s;
    position: absolute;
    transform: ${isVisible ? 'translateY(-5%)' : 'translateY(20%)'} ;
    left: -5%;
    height: 100%;
    max-width: 33.3%;
  `};

  ${media.greaterThan('xl')`
    position: absolute;
    left: -5%;
    height: 100%;
    max-width: 33.3%;
  `};
`
export const SecondCard = styled.article<{
  isVisible: boolean
}>`
  ${({ isVisible }) => media.greaterThan('lg')`
      transition: all 1s;
      transition-delay: 0.5s;
      position: absolute;
      left: 29%;
      transform: ${isVisible ? 'translateY(5%)' : 'translateY(20%)'} ;
      height: 100%;
      max-width: 33.3%;
  `};

  ${media.greaterThan('xl')`
      position: absolute;
      left: 29%;
      height: 100%;
      max-width: 33.3%;
  `};
`
export const ThirdCard = styled.article<{
  isVisible: boolean
}>`
  ${({ isVisible }) => media.greaterThan('lg')`
      transition: all 1s;
      transition-delay: 0.7s;
      position: absolute;
      left: 63.4%;
      transform: ${isVisible ? 'translateY(15%)' : 'translateY(25%)'} ;
      height: 100%;
      max-width: 33.3%;
  `};

  ${media.greaterThan('xl')`
      position: absolute;
      left: 63.4%;
      height: 100%;
      max-width: 33.3%;
  `};
`
