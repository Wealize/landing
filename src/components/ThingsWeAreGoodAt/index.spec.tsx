import React from 'react'
import { mount } from 'enzyme'

import { Title, FirstCard, SecondCard, ThirdCard } from './styles'
import ThingsWeAreGoodAt from './index'

describe('<ThingsWeAreGoodAt />', () => {
  const wrapper = mount(<ThingsWeAreGoodAt />)

  it('should render', () => {
    expect(wrapper).toBeDefined()
  })

  it('should render content correctly', () => {
    expect(wrapper.find(Title).text().length).toBeGreaterThan(0)
    expect(wrapper.find(FirstCard).length).toEqual(1)
    expect(wrapper.find(SecondCard).length).toEqual(1)
    expect(wrapper.find(ThirdCard).length).toEqual(1)
  })
})
