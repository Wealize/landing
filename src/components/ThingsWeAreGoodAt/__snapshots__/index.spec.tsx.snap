// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ThingsWeAreGoodAt /> should render 1`] = `
<Memo(ThingsWeAreGoodAt)>
  <styles__BgContainer>
    <div
      className="styles__BgContainer-sc-3xtj1d-0 cPYCuj"
    >
      <styles__Container>
        <div
          className="styles__Container-sc-3xtj1d-1 gosfSK"
        >
          <styles__Title>
            <h3
              className="styles__Title-sc-3xtj1d-2 bCWcjP"
            >
              profile:things-we-are-good-at.title
            </h3>
          </styles__Title>
          <styles__CardsContainer>
            <div
              className="styles__CardsContainer-sc-3xtj1d-3 gdwoNh"
            >
              <styles__FirstCard>
                <article
                  className="styles__FirstCard-sc-3xtj1d-4 CuGla"
                >
                  <Memo(ExpandableCard)
                    backgroundColor="#ffc9c9"
                    description="profile:things-we-are-good-at.first-card.description"
                    headline="profile:things-we-are-good-at.first-card.headline"
                    image={<Memo(BlockchainIcon) />}
                    isExpanded={true}
                    subheadline="profile:things-we-are-good-at.first-card.subheadline"
                    textColor="#f06078"
                  >
                    <styles__Container
                      backgroundColor="#ffc9c9"
                      className="ExpandableCard"
                      isExpanded={true}
                      textColor="#f06078"
                    >
                      <div
                        className="styles__Container-qa6j6v-0 lhVvjX ExpandableCard"
                      >
                        <styles__Header
                          isExpanded={true}
                        >
                          <div
                            className="styles__Header-qa6j6v-1 jeKrYK"
                          >
                            <styles__Headline>
                              <h3
                                className="styles__Headline-qa6j6v-3 ewkpmO"
                              >
                                profile:things-we-are-good-at.first-card.headline
                              </h3>
                            </styles__Headline>
                            <styles__Toggle
                              aria-label="expand card content button"
                              isExpanded={true}
                              onClick={[Function]}
                              textColor="#f06078"
                            >
                              <button
                                aria-label="expand card content button"
                                className="styles__Toggle-qa6j6v-4 oOZiH"
                                onClick={[Function]}
                              >
                                <Memo(RoundArrowIcon)>
                                  <svg
                                    height="100%"
                                    viewBox="0 0 40 40"
                                    width="100%"
                                    xmlns="http://www.w3.org/2000/svg"
                                  >
                                    <g
                                      fill="none"
                                    >
                                      <path
                                        d="M20 40a20 20 0 1120-20 20 20 0 01-20 20z"
                                      />
                                      <path
                                        d="M20 38.5c4.942 0 9.587-1.924 13.081-5.419A18.375 18.375 0 0038.5 20c0-4.942-1.924-9.587-5.419-13.081A18.375 18.375 0 0020 1.5 18.374 18.374 0 006.919 6.919 18.375 18.375 0 001.5 20c0 4.942 1.924 9.587 5.419 13.081A18.375 18.375 0 0020 38.5m0 1.5C8.954 40 0 31.046 0 20S8.954 0 20 0s20 8.954 20 20-8.954 20-20 20z"
                                        fill="#f06078"
                                      />
                                    </g>
                                    <path
                                      d="M13.79 15.465l5.4-4.139v19.673h1.5V11.325l5.4 4.139.912-1.19-6.311-4.839-.045-.035h-1.41l-.045.034-6.312 4.839z"
                                      fill="#f06078"
                                    />
                                  </svg>
                                </Memo(RoundArrowIcon)>
                              </button>
                            </styles__Toggle>
                          </div>
                        </styles__Header>
                        <styles__Content
                          isExpanded={true}
                        >
                          <div
                            className="styles__Content-qa6j6v-2 gkxgYk"
                          >
                            <styles__ImageContainer>
                              <div
                                className="styles__ImageContainer-qa6j6v-5 kNExPN"
                              >
                                <Memo(BlockchainIcon)>
                                  <svg
                                    height="100%"
                                    viewBox="0 0 280 280"
                                    width="100%"
                                    xmlns="http://www.w3.org/2000/svg"
                                  >
                                    <title>
                                      blockchain@1x
                                    </title>
                                    <g
                                      fill="none"
                                      fillRule="evenodd"
                                    >
                                      <g
                                        fill="#F06078"
                                        fillRule="nonzero"
                                      >
                                        <path
                                          d="M84.976 87.703H46.361V49.06h38.615z"
                                          style={
                                            Object {
                                              "mixBlendMode": "multiply",
                                            }
                                          }
                                        />
                                        <path
                                          d="M117.68 113.972H65.668v-52.05h52.012z"
                                        />
                                        <path
                                          d="M157.667 154.178H91.675v-66.04h65.992z"
                                          style={
                                            Object {
                                              "mixBlendMode": "multiply",
                                            }
                                          }
                                        />
                                        <path
                                          d="M236.033 232.482H124.67V121.04h111.363z"
                                        />
                                      </g>
                                      <path
                                        d="M91.674 88.139h26.006V114H91.674z"
                                        fill="#E12438"
                                      />
                                    </g>
                                  </svg>
                                </Memo(BlockchainIcon)>
                              </div>
                            </styles__ImageContainer>
                            <styles__Subheadline>
                              <h4
                                className="styles__Subheadline-qa6j6v-6 dgfMQj"
                              >
                                profile:things-we-are-good-at.first-card.subheadline
                              </h4>
                            </styles__Subheadline>
                            <styles__Description>
                              <h5
                                className="styles__Description-qa6j6v-7 gWlZeG"
                              >
                                profile:things-we-are-good-at.first-card.description
                              </h5>
                            </styles__Description>
                          </div>
                        </styles__Content>
                      </div>
                    </styles__Container>
                  </Memo(ExpandableCard)>
                </article>
              </styles__FirstCard>
              <styles__SecondCard>
                <article
                  className="styles__SecondCard-sc-3xtj1d-5 hDLePU"
                >
                  <Memo(ExpandableCard)
                    backgroundColor="#f06078"
                    description="profile:things-we-are-good-at.second-card.description"
                    headline="profile:things-we-are-good-at.second-card.headline"
                    image={<Memo(CognitiveIcon) />}
                    isExpanded={false}
                    subheadline="profile:things-we-are-good-at.second-card.subheadline"
                    textColor="#ffffff"
                  >
                    <styles__Container
                      backgroundColor="#f06078"
                      className="ExpandableCard"
                      isExpanded={false}
                      textColor="#ffffff"
                    >
                      <div
                        className="styles__Container-qa6j6v-0 dJhQvX ExpandableCard"
                      >
                        <styles__Header
                          isExpanded={false}
                        >
                          <div
                            className="styles__Header-qa6j6v-1 WstWs"
                          >
                            <styles__Headline>
                              <h3
                                className="styles__Headline-qa6j6v-3 ewkpmO"
                              >
                                profile:things-we-are-good-at.second-card.headline
                              </h3>
                            </styles__Headline>
                            <styles__Toggle
                              aria-label="expand card content button"
                              isExpanded={false}
                              onClick={[Function]}
                              textColor="#ffffff"
                            >
                              <button
                                aria-label="expand card content button"
                                className="styles__Toggle-qa6j6v-4 euQgEa"
                                onClick={[Function]}
                              >
                                <Memo(RoundArrowIcon)>
                                  <svg
                                    height="100%"
                                    viewBox="0 0 40 40"
                                    width="100%"
                                    xmlns="http://www.w3.org/2000/svg"
                                  >
                                    <g
                                      fill="none"
                                    >
                                      <path
                                        d="M20 40a20 20 0 1120-20 20 20 0 01-20 20z"
                                      />
                                      <path
                                        d="M20 38.5c4.942 0 9.587-1.924 13.081-5.419A18.375 18.375 0 0038.5 20c0-4.942-1.924-9.587-5.419-13.081A18.375 18.375 0 0020 1.5 18.374 18.374 0 006.919 6.919 18.375 18.375 0 001.5 20c0 4.942 1.924 9.587 5.419 13.081A18.375 18.375 0 0020 38.5m0 1.5C8.954 40 0 31.046 0 20S8.954 0 20 0s20 8.954 20 20-8.954 20-20 20z"
                                        fill="#f06078"
                                      />
                                    </g>
                                    <path
                                      d="M13.79 15.465l5.4-4.139v19.673h1.5V11.325l5.4 4.139.912-1.19-6.311-4.839-.045-.035h-1.41l-.045.034-6.312 4.839z"
                                      fill="#f06078"
                                    />
                                  </svg>
                                </Memo(RoundArrowIcon)>
                              </button>
                            </styles__Toggle>
                          </div>
                        </styles__Header>
                        <styles__Content
                          isExpanded={false}
                        >
                          <div
                            className="styles__Content-qa6j6v-2 jLwxFh"
                          >
                            <styles__ImageContainer>
                              <div
                                className="styles__ImageContainer-qa6j6v-5 kNExPN"
                              >
                                <Memo(CognitiveIcon)>
                                  <svg
                                    height="100%"
                                    viewBox="0 0 280 280"
                                    width="100%"
                                    xmlns="http://www.w3.org/2000/svg"
                                  >
                                    <title>
                                      cognitives@1x
                                    </title>
                                    <g
                                      fill="#FFC9C9"
                                      fillRule="nonzero"
                                      transform="translate(-1 94)"
                                    >
                                      <path
                                        d="M137.098 0h92.046v92.047h-92.046z"
                                      />
                                      <path
                                        d="M45.985 0h92.046v92.047H45.985z"
                                      />
                                      <path
                                        d="M229.144.001l-53.143 92.046h106.286z"
                                        style={
                                          Object {
                                            "mixBlendMode": "multiply",
                                          }
                                        }
                                      />
                                      <circle
                                        cx={45.985}
                                        cy={46.024}
                                        r={45.985}
                                        style={
                                          Object {
                                            "mixBlendMode": "multiply",
                                          }
                                        }
                                      />
                                    </g>
                                  </svg>
                                </Memo(CognitiveIcon)>
                              </div>
                            </styles__ImageContainer>
                            <styles__Subheadline>
                              <h4
                                className="styles__Subheadline-qa6j6v-6 dgfMQj"
                              >
                                profile:things-we-are-good-at.second-card.subheadline
                              </h4>
                            </styles__Subheadline>
                            <styles__Description>
                              <h5
                                className="styles__Description-qa6j6v-7 gWlZeG"
                              >
                                profile:things-we-are-good-at.second-card.description
                              </h5>
                            </styles__Description>
                          </div>
                        </styles__Content>
                      </div>
                    </styles__Container>
                  </Memo(ExpandableCard)>
                </article>
              </styles__SecondCard>
              <styles__ThirdCard>
                <article
                  className="styles__ThirdCard-sc-3xtj1d-6 gqpYNn"
                >
                  <Memo(ExpandableCard)
                    backgroundColor="#8fd0c8"
                    description="profile:things-we-are-good-at.third-card.description"
                    headline="profile:things-we-are-good-at.third-card.headline"
                    image={<Memo(ConversationalIcon) />}
                    isExpanded={false}
                    subheadline="profile:things-we-are-good-at.third-card.subheadline"
                    textColor="#6666ff"
                  >
                    <styles__Container
                      backgroundColor="#8fd0c8"
                      className="ExpandableCard"
                      isExpanded={false}
                      textColor="#6666ff"
                    >
                      <div
                        className="styles__Container-qa6j6v-0 fzkjnN ExpandableCard"
                      >
                        <styles__Header
                          isExpanded={false}
                        >
                          <div
                            className="styles__Header-qa6j6v-1 WstWs"
                          >
                            <styles__Headline>
                              <h3
                                className="styles__Headline-qa6j6v-3 ewkpmO"
                              >
                                profile:things-we-are-good-at.third-card.headline
                              </h3>
                            </styles__Headline>
                            <styles__Toggle
                              aria-label="expand card content button"
                              isExpanded={false}
                              onClick={[Function]}
                              textColor="#6666ff"
                            >
                              <button
                                aria-label="expand card content button"
                                className="styles__Toggle-qa6j6v-4 jCmDwW"
                                onClick={[Function]}
                              >
                                <Memo(RoundArrowIcon)>
                                  <svg
                                    height="100%"
                                    viewBox="0 0 40 40"
                                    width="100%"
                                    xmlns="http://www.w3.org/2000/svg"
                                  >
                                    <g
                                      fill="none"
                                    >
                                      <path
                                        d="M20 40a20 20 0 1120-20 20 20 0 01-20 20z"
                                      />
                                      <path
                                        d="M20 38.5c4.942 0 9.587-1.924 13.081-5.419A18.375 18.375 0 0038.5 20c0-4.942-1.924-9.587-5.419-13.081A18.375 18.375 0 0020 1.5 18.374 18.374 0 006.919 6.919 18.375 18.375 0 001.5 20c0 4.942 1.924 9.587 5.419 13.081A18.375 18.375 0 0020 38.5m0 1.5C8.954 40 0 31.046 0 20S8.954 0 20 0s20 8.954 20 20-8.954 20-20 20z"
                                        fill="#f06078"
                                      />
                                    </g>
                                    <path
                                      d="M13.79 15.465l5.4-4.139v19.673h1.5V11.325l5.4 4.139.912-1.19-6.311-4.839-.045-.035h-1.41l-.045.034-6.312 4.839z"
                                      fill="#f06078"
                                    />
                                  </svg>
                                </Memo(RoundArrowIcon)>
                              </button>
                            </styles__Toggle>
                          </div>
                        </styles__Header>
                        <styles__Content
                          isExpanded={false}
                        >
                          <div
                            className="styles__Content-qa6j6v-2 jLwxFh"
                          >
                            <styles__ImageContainer>
                              <div
                                className="styles__ImageContainer-qa6j6v-5 kNExPN"
                              >
                                <Memo(ConversationalIcon)>
                                  <svg
                                    height="100%"
                                    viewBox="0 0 280 280"
                                    width="100%"
                                    xmlns="http://www.w3.org/2000/svg"
                                  >
                                    <title>
                                      conversational@1x
                                    </title>
                                    <g
                                      fill="#66F"
                                      fillRule="nonzero"
                                      transform="translate(34 46)"
                                    >
                                      <circle
                                        cx={59.263}
                                        cy={128.993}
                                        r={59.263}
                                      />
                                      <path
                                        d="M59.269 24.45h118.526v118.526H59.269z"
                                        style={
                                          Object {
                                            "mixBlendMode": "multiply",
                                          }
                                        }
                                      />
                                      <circle
                                        cx={177.79}
                                        cy={35.119}
                                        r={35.119}
                                      />
                                    </g>
                                  </svg>
                                </Memo(ConversationalIcon)>
                              </div>
                            </styles__ImageContainer>
                            <styles__Subheadline>
                              <h4
                                className="styles__Subheadline-qa6j6v-6 dgfMQj"
                              >
                                profile:things-we-are-good-at.third-card.subheadline
                              </h4>
                            </styles__Subheadline>
                            <styles__Description>
                              <h5
                                className="styles__Description-qa6j6v-7 gWlZeG"
                              >
                                profile:things-we-are-good-at.third-card.description
                              </h5>
                            </styles__Description>
                          </div>
                        </styles__Content>
                      </div>
                    </styles__Container>
                  </Memo(ExpandableCard)>
                </article>
              </styles__ThirdCard>
            </div>
          </styles__CardsContainer>
        </div>
      </styles__Container>
    </div>
  </styles__BgContainer>
</Memo(ThingsWeAreGoodAt)>
`;
